install/0000_90_cluster-version-operator_02_servicemonitor.yaml

```diff
@@ -97,6 +102,7 @@ spec:
         ) == 1
       for: 30m
       labels:
+        namespace: openshift-cluster-version
```

Comment:
`ClusterOperatorDegraded` already has `namespace` in Telemetry (this time via `cluster_operator_conditions`, so I'd rather drop this line from the commit as redundant.  We might need to adjust the `cluster_operator_up` to preserve `namespace` in that case.  Something like:

```
cluster_operator_conditions{job="cluster-version-operator", condition="Degraded"}
or on (namespace, name)
group by (namespace, name) (cluster_operator_up{job="cluster-version-operator"})
```

Reply:
Thank you, fixed in https://github.com/openshift/cluster-version-operator/pull/800/commits/d525eb97a2c971b4837210963d86d3b617452891.